src/Package/Impl/Shell/VsAppShell.cs

```diff
@@ -394,6 +399,19 @@ public sealed class VsAppShell : IApplicationShell, IMainThread, IIdleTimeServic
                 }
             }
         }
+
+        public IWritableSettingsStorage SettingsStorage {
```

Comment:
```REditorSettings``` uses it to store stuff, but it is named/typed, contains commonly used stuff exposed to user somewhere. This one is low level, anyone can use it to store things. Originally I imported it directly, but @int19h suggested to have a shell member instead.